LoRaEngine/deployment.ci.aio.template.json

```diff
@@ -68,13 +68,22 @@
               "image": "${MODULES.LoRaWanNetworkSrvModule}",
               "createOptions": {
                 "ExposedPorts": {
-                  "5000/tcp": {}
+                  "5001/tcp": {},
+                  "5002/tcp": {}
                 },
                 "HostConfig": {
+                  "Binds":  [
+                    "$CERT_REMOTE_PATH:/var/lorastarterkit/certs/"
+                  ],
                   "PortBindings": {
-                    "5000/tcp": [
+                    "5001/tcp": [
```

Comment:
@bastbu Do we need to change something on the metrics collector? I remember this listen on 5000? Would we also need to generate a cert for it?

Reply:
#1025 has been created for checking the behavior